spec/requests/admin/podcasts_spec.rb

```diff
@@ -62,15 +62,56 @@
   describe "Updating" do
     it "updates" do
       put admin_podcast_path(podcast), params: { podcast: { title: "hello",
-                                                            feed_url: "https://pod.example.com/rss.rss" } }
+                                                            feed_url: "https://pod.example.com/rss.rss",
+                                                            description: "Description",
+                                                            itunes_url: "https://itunes.example.com",
+                                                            overcast_url: "https://overcast.example.com",
+                                                            android_url: "https://android.example.com",
+                                                            soundcloud_url: "https://soundcloud.example.com",
+                                                            website_url: "https://example.com",
+                                                            twitter_username: "@ThePracticalDev",
+                                                            pattern_image: "https://dummyimage.com/400x400.jpg",
+                                                            main_color_hex: "#ffffff",
+                                                            slug: "postcast-test-url",
+                                                            image: "https://dummyimage.com/400x400.jpg",
+                                                            reachable: true,
+                                                            published: true
+                                                          } }
       podcast.reload
       expect(podcast.title).to eq("hello")
       expect(podcast.feed_url).to eq("https://pod.example.com/rss.rss")
+      expect(podcast.description).to eq("Description")
+      expect(podcast.itunes_url).to eq("https://itunes.example.com")
+      expect(podcast.overcast_url).to eq("https://overcast.example.com")
+      expect(podcast.android_url).to eq("https://android.example.com")
+      expect(podcast.soundcloud_url).to eq("https://soundcloud.example.com")
+      expect(podcast.website_url).to eq("https://example.com")
+      expect(podcast.twitter_username).to eq("@ThePracticalDev")
+      expect(podcast.pattern_image).to eq("https://dummyimage.com/400x400.jpg")
+      expect(podcast.main_color_hex).to eq("#ffffff")
+      expect(podcast.slug).to eq("postcast-test-url")
+      expect(podcast.image).to eq("https://dummyimage.com/400x400.jpg")
```

Comment:
Since a separate test would validate the images being updated there's no need to check for them here. Also important to update the expected value for `main_color_hex`

```suggestion
      expect(podcast.main_color_hex).to eq("ffffff")
      expect(podcast.slug).to eq("postcast-test-url")
```